handlers/main.yml

```diff
@@ -48,4 +49,10 @@
   delay: 3
   register: result
   until: result is success
+
+- name: Restart rc-local
+  service:
+    name: rc-local
+    state: restarted
+  become: yes
```

Comment:
Here be dragons - what if rc.local had other things in it too (as it easily could).  Perhaps defining a systemd unit for those tuning parameters would be safer.

Reply:
I have moved these jobs over to systemd unit files.